tests/manage/storageclass/test_create_multiple_sc_with_different_pool_name.py

```diff
@@ -70,6 +69,8 @@ def test_create_multiple_sc_with_different_pool_name(
                 f"PVC: {pvc_obj.name} created successfully using "
                 f"{sc_list[i].name}"
             )
+            helpers.wait_for_resource_state(pvc_obj, constants.STATUS_BOUND)
```

Comment:
In case the PVC with not reach Bound status, you will not have pvc_obj in the teardown factory.
You should add it to teardown_factory right after its creation and continue to do all other things

Reply:
Done